### ISSUE TITLE
Bug Feedback for Wraith King's Wraithfire Blast

### ISSUE THREAD

**Issue by misasuka** [ValveSoftware/Dota2-Gameplay#17798]
### Ability name

Wraithfire Blast

### Description

I have noticed an issue with Wraith King's Wraithfire Blast when it reaches level 4. Specifically, the sustained damage from the burn effect seems to be sometimes (not always) missing one instance of damage. This inconsistency affects the overall damage output and may impact gameplay balance. Please review and address this issue.

https://github.com/ValveSoftware/Dota2-Gameplay/assets/170601435/f062b302-297e-4891-b623-811f58fbf38e

https://github.com/ValveSoftware/Dota2-Gameplay/assets/170601435/9dd1dbb3-74b3-43bf-ba65-96ac0c17e3a6

### Example Match ID (and possibly Timestamp)

see the videos

**Comment by gwtjxyz** [comment]
yes, just noticed this as well. Doesn't happen just with level 4, I've had it happen with the level 2 blast in my tests as well. It seems like it happens irrespective of levels depending on which tick you launched the blast on. Here's a combat log screenshot of it happening to a level 4 wraithfire blast:

![dota2_24-05-2024-18-11-51](https://github.com/ValveSoftware/Dota2-Gameplay/assets/20083177/37650ecc-affc-48a8-9744-348d42ae7654)
